src/Common/test-runtime/project.json

```diff
@@ -1,9 +1,9 @@
 {
   "dependencies": {
-    "Microsoft.NETCore.Platforms": "1.0.1-rc2-23610",
-    "Microsoft.NETCore.TestHost": "1.0.0-rc2-23610",
-    "Microsoft.NETCore.Console": "1.0.0-rc2-23610",
-    "System.IO.Compression": "4.1.0-rc2-23610",
+    "Microsoft.NETCore.Platforms": "1.0.1-rc2-23616",
```

Comment:
The package validation task will fall if you only update this one project to the new versions. You will need to update the entire repo. See https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/project-nuget-dependencies.md#upgrading-a-package-dependency